roles/openshift_common/tasks/main.yml

```diff
@@ -22,6 +22,7 @@
       deployment_type: "{{ openshift_deployment_type }}"
       use_fluentd: "{{ openshift_use_fluentd | default(None) }}"
       use_flannel: "{{ openshift_use_flannel | default(None) }}"
+      use_manageiq: "{{ use_manageiq | default(None) }}"
```

Comment:
this should probably be:
```
      use_manageiq: "{{ openshift_use_manageiq | default(None) }}"
```

We attempt to prefix variables that are meant to be set in the inventory (or group_vars/host_vars) to avoid issues with ansible variable scoping.